docs/cloud/manage/role-based-access.md

```diff
@@ -0,0 +1,139 @@
+| Appoint Administrators | :heavy_check_mark: | - | - | - | - | |
+| Appoint Billing user | :heavy_check_mark: | - | - | - | - | |
+| Appoint Managers | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| Appoint Troubleshooters | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| Appoint Observer | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| See all Users in a Room | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | |
+| Invite existing user to Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | User already invited to the Space |
+| Remove user from Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+
+##### Room Management
+
+| **Functionality** | **Administrator** | **Manager** | **Troubleshooter** | **Observer** | **Billing** | Notes |
+| :-- | :--: | :--: | :--: | :--: | :--: | :-- |
+| See all Rooms in a Space | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | - | |
+| Join any Room in a Space | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | - | By joining a room you will be enabled to get notifications from nodes on that room |
+| Leave Room  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | |
+| Create a new Room in a Space | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| Delete Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| Change Room name | :heavy_check_mark: | :heavy_check_mark: | - | - | - | If not the _All Nodes_ room |
+| Change Room description | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+| Add existing Nodes to Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | Node already connected to the Space |
+| Remove Nodes from Room | :heavy_check_mark: | :heavy_check_mark: | - | - | - | |
+
+##### Notifications Management
+
+| **Functionality** | **Administrator** | **Manager** | **Troubleshooter** | **Observer** | **Billing** | Notes |
+| :-- | :--: | :--: | :--: | :--: | :--: | :-- |
+| See all configured notifications on a Space  | :heavy_check_mark: | - | - | - | - | |
+| Add new configuration | :heavy_check_mark: | - | - | - | - | |
+| Enable/Disable configuration | :heavy_check_mark: | - | - | - | - |  |
+| Edit configuration | :heavy_check_mark: | - | - | - | - | Some exceptions apply depending on [service level](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md#available-actions-per-notification-methods-based-on-service-level) |
+| Delete configuration | :heavy_check_mark: | - | - | - | - | |
+| Edit personal level notification settings | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [Manage user notification settings](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md#manage-user-notification-settings) |
```

Comment:
I think `billing` cannot change their notification settings since this role doesn't have any access to nodes thus to alerts as well.

Reply:
thanks for the feedback @iorvd , that makes sense and this was actually with a `?` on the google sheet if that is the implementation we should udpate the google sheet
we also had some other `? ` on the definition of roles can you tell what was the implementation?
![image](https://user-images.githubusercontent.com/82235632/217024334-1fbf7917-8e05-46c2-8290-7e8a0019617c.png)


![image](https://user-images.githubusercontent.com/82235632/217024210-4241bc35-8b10-4e8a-8516-d5c6401f9dc9.png)

Reply:
Personal user settings do not check permissions at all, so I guess they're all `x`